salt/orchestrate/aws/cloud_profiles/sandbox.conf

```diff
@@ -0,0 +1,21 @@
+# -*- mode: yaml; coding: utf-8; -*-
+sandbox:
+  provider: mitx
+  size: t2.medium
+  image: {{ salt.sdb.get('sdb://consul/xenial_ami_id') }}
+  ssh_username: ubuntu
+  ssh_interface: private_ips
+  block_device_mappings:
+    - DeviceName: /dev/sda1
+      Ebs.VolumeSize: 8
```

Comment:
Might want to increase the disk size for this instance since it will be containing some course data